api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/subscriptions_controller.rb

```diff
@@ -102,7 +102,7 @@ def fulfill
   def subscription_update_params
     # custom_data is an arbitrary hash
     # On Rails 5.1 use `permit(custom_data: {})`
-    params.require(:subscription).permit(:start_date, :max_licenses, :custom_data).tap do |whitelisted|
+    params.require(:subscription).permit(:start_date, :max_licenses, :custom_data, :product_pricing_id).tap do |whitelisted|
```

Comment:
Should we also whitelist `product_contract_id` here? I'm thinking it's something we should be able to update since the previous code was setting a relationship for it.

Reply:
Yes!